src/client/Root.js

```diff
@@ -1,15 +1,10 @@
 import React from 'react';
-import { createStore } from 'redux';
 import { Provider } from 'react-redux';
-import indexReducer from './redux/reducers';
 
-const Root = ({ children, initialState = { } }) => {
-    const store = createStore(indexReducer, initialState);
-    return (
-        <Provider store={ store }>
-            { children }
-        </Provider>
-    );
-};
+import store from './redux';
+
+const Root = ({ children }) => <Provider store={ store }>
+{ children }
+</Provider>;
```

Comment:
Can you please convert this too class component. Cause as this is mostly used by beginners and this syntax would be lil confusing to them. I always tries to keep these things in classful components if those are shared to others and have something to do with states.
I guess this changed in one of the commit in last refactoring PR.